### PR TITLE
Generate catalog/subscriptions from olm bundles for openshift

### PR DESCRIPTION
**Pull request by adwk67**
Adds a new script for combining operator bundles into a catalog, with the option of deploying operator subscriptions.

### CODE REVIEW

File: olm/build-catalog.sh [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  setup
+  prerequisites
+  catalog
+
+  # install catalog/group for all operators
+  kubectl apply -f catalog-source.yaml
+  kubectl apply -f operator-group.yaml
+
+  subscriptions
```
Comment (razvan) [inline olm/build-catalog.sh:125]: I think, this should go in a separate `deploy() `function that only runs on demand (based on a command line arg).

File: olm/catalog-source.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: stackable-operators-catalog
+  namespace: stackable-operators
+spec:
+  sourceType: grpc
+  image: docker.stackable.tech/sandbox/test/stackable-operators-catalog:23.1.0
```
Comment (razvan) [inline olm/catalog-source.yaml:9]: the version is hard coded here.

File: olm/scc.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: zookeeper-scc
```
Comment (razvan) [inline olm/scc.yaml:5]: Currently all custom SCCs are the same. Maybe rename to `stackable-platform-scc` and use it where needed?

**Review by razvan — APPROVED** [review]
It looks good to me as is, but see comments below.

File: olm/catalog-source.yaml (hunk 2 above)
Comment (adwk67) [reply olm/catalog-source.yaml:9]: Done.

File: olm/build-catalog.sh (hunk 1 above)
Comment (adwk67) [reply olm/build-catalog.sh]: Done.

File: olm/scc.yaml (hunk 3 above)
Comment (adwk67) [reply olm/scc.yaml]: Done.

**Review by razvan — APPROVED** [review]

**Pull request merged**